Client/Ecosia/UI/NTP/Impact/NTPImpactRowView.swift

```diff
@@ -45,14 +44,17 @@ final class NTPImpactRowView: UIView, NotificationThemeable {
     private lazy var subtitleLabel: UILabel = {
         let label = UILabel()
         label.font = .preferredFont(forTextStyle: .footnote)
-        label.adjustsFontSizeToFitWidth = true
+        label.numberOfLines = 0
         return label
     }()
-    private lazy var actionButton: UIButton = {
-        let button = UIButton()
+    private lazy var actionButton: ResizableButton = {
+        let button = ResizableButton()
         button.translatesAutoresizingMaskIntoConstraints = false
         button.titleLabel?.font = .preferredFont(forTextStyle: .callout)
-        button.titleLabel?.adjustsFontSizeToFitWidth = true
+        button.titleLabel?.textAlignment = .right
```

Comment:
Will this affect RTL languages?

Reply:
That's a good question, just tried it with Hebrew and I think it stays right aligned, which seems to actually look better since the button is on the left side and we don't have the translation for it anyway.

Even so, no idea if this is actually accessible then, my guess would be no since we do not translate 🤔 

Would not spend much more time on it now, but if you have a quick suggestion please let me know 🙂

Reply:
Better to review it separately. As it's `textAlignment` and we shouldn't force the direction unless strictly needed.